CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ Changelog
 
 ## vNext
 
+### Features
+
+- Add internal link support in newsfeed items ([PR 1617](https://github.com/input-output-hk/daedalus/pull/1617))
```

Comment:
Maybe add "hash verification", as it's part of its scope?

Reply:
@daniloprates 
`Add internal link support in newsfeed items and verification hash generator script`
Ok?